CustomUIButton.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomUIButton.swift
+//  Connnect
+//
+//  Created by Stella Su on 11/22/16.
+//  Copyright © 2016 iOS-Connect. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CustomUIButton: UIButton{
```

Comment:
This looks super 👍 🎉 👍 💯 

My only suggestion would be to give it a more descriptive name. Maybe something like `BorderedButton` or `RoundedButton` or maybe just `Button`